lib/bap_ida/bap_ida.mli

```diff
@@ -26,7 +26,7 @@ module Std : sig
 
     (** [create target] create an IDA instance that will work with
         [target] executable. *)
-    val create : string -> t
```

Comment:
Bgg... I was sure that I reverted it! Anyway, I the PR is still very weak, made it for the discussion purpose. There are still issues that we must resolve. Btw I was unable to introduce builtin types on a lexer level, so some work still should be done on the ida side.